src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/Views/List/List.jsx

```diff
@@ -47,6 +47,14 @@ const columnConfig = (t, showSubContentsCount) => {
     return columns;
 };
 
+const columnIdFieldNameMapper = {
```

Comment:
```suggestion
const columnFieldNameMapper = {
```

Reply:
On one side we have the columnId and on the other side we have the fieldName, so I won't take your suggestion